src/SFA.DAS.Commitments.Api.Client/CommitmentsApi.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using SFA.DAS.Commitments.Api.Types;
+
+namespace SFA.DAS.Commitments.Api.Client
+{
+    public class CommitmentsApi : ICommitmentsApi
+    {
+        private readonly string _baseUrl;
+
+        public CommitmentsApi(string baseUrl)
+        {
+            if (baseUrl == null)
+                throw new ArgumentNullException(nameof(baseUrl));
+            _baseUrl = baseUrl;
+        }
+
+        public async Task<List<CommitmentListItem>> GetProviderCommitments(long providerId)
+        {
+            var url = $"{_baseUrl}api/provider/{providerId}/commitments";
```

Comment:
Are we having a different url for a provider and a employer or are we basing on the token contents?